alerter/alerter.java

```diff
@@ -1,27 +1,11 @@
-public class Alerter {
-    static int alertFailureCount = 0;
-    static int networkAlertStub(float celcius) {
-        System.out.println("ALERT: Temperature is " + celcius + " celcius");
-        // Return 200 for ok
-        // Return 500 for not-ok
-        // stub always succeeds and returns 200
-        return 200;
-    }
-    static void alertInCelcius(float farenheit) {
-        float celcius = (farenheit - 32) * 5 / 9;
-        int returnCode = networkAlertStub(celcius);
+public class alerter {
+    static int failedCount = 0;
+    static void alertInCelsius(float fahrenheit) {
+        RequestAlert alert = new AlertNetwork();

```

Comment:
Alert Network class is a test class that is used as stub in the current implementation. the instantiation of this object should be done outside of production code, suitably in main, to keep production code unaware of test code.